common/config/config.go

```diff
@@ -119,6 +119,16 @@ func GetStateHashCheckHeight(id uint32) uint32 {
 	return STATE_HASH_CHECK_HEIGHT[id]
 }
 
+var CROSS_CHAIN_CHECK_HEIGHT = map[uint32]uint32{
+	NETWORK_ID_MAIN_NET:    constants.CROSS_CHAIN_HEIGHT_MAINNET,
```

Comment:
no cross chain message in history block. so this is not necessary.

Reply:
sure?

Reply:
if some one send the transaction to attack you, it maybe have some error.

Reply:
remove